docs/core/porting/versioning-sdk-msbuild-vs.md

```diff
@@ -51,6 +51,8 @@ The support timeframe for the SDK typically matches that of the Visual Studio ve
 > <sup>4</sup> 6.0.300 and newer SDKs require a minimum Visual Studio version of 17.0.
 >
 > <sup>5</sup> 7.0.200 and newer SDKs require a minimum Visual Studio version of 17.4.
+> 
+> <sup>6</sup> Starting with .NET 6, .1xx SDK feature band is supported throughout the lifecycle of major .NET versions. During the extended support period, support is limited to security fixes and minimal high-priority non-security fixes for Linux only. To learn more about the reasoning for this extended support, refer to this [document](https://github.com/dotnet/source-build#support).
```

Comment:
It doesn't feel necessary to say `Starting with .NET 6` because as of now all in-support versions have this support policy.  Over time it will become meaningless to start with this.